sssd_test_framework/misc/__init__.py

```diff
@@ -1,4 +1,5 @@
 """Miscellaneous functions."""
+
```

Comment:
Looks like your IDE is updating this file?

Reply:
Yes,  this was the same as the hosts/__init__.py.   Let's get an answer there before I drop this one.

Reply:
dropped the commit